packages/shared/trade/src/deposit-input/deposit-input.tsx

```diff
@@ -50,7 +49,6 @@ export const DepositInput = React.forwardRef<
       isWithdraw,
       maxWithdraw,
       useZeroDefault,
-      requiredApprovalAmount,
```

Comment:
I think we're losing some bugfix for an edge case here by removing it, it has to do with debt repayment...

Reply:
It's not being used anywhere in the component though

Reply:
Yes, it needs to be passed into the useTransactionApprovals hook instead